docs/UserGuide.md

```diff
@@ -290,10 +290,11 @@ Filters tasks in the task list by various predicates:
 Format: `filter (ta/TASK_ASSIGNEE_NAME)||(td/DEADLINE)||(start/START_DATE end/END_DATE)||(tp/TASK PROGRESS)||(tn/TASK_NAME)` 
   - User may choose one predicate to filter tasks by
   - Assignee name is the name of the Teammate who is assigned to the task
+  - Task Name can be any alphanumeric value (containing only alphabets and / or numbers).
+  - Names provided must be complete words and will only match complete words from the name of the task or assignee. 
```

Comment:
names or name? word or words?

Reply:
Yes, names refer to both task name and assignee name. Words as you can have more than 1 keyword